runelite-client/src/main/java/net/runelite/client/plugins/chathistory/ChatHistoryPlugin.java

```diff
@@ -57,22 +73,53 @@
 		ChatMessageType.GAME
 	);
 
+	private static final int CYCLE_HOTKEY = KeyEvent.VK_TAB;
+	private static final Set<ChatMessageType> CYCLE_TYPES = Sets.newHashSet(
```

Comment:
`ImmutableSet`

Reply:
I'll update the one right above this as well then.